docs/README.adoc

```diff
@@ -0,0 +1,99 @@
+= ShiftStack CI
+
+Each OpenShift on OpenStack continuous integration test job runs on one of two clouds:
+
+* MassOpenCloud: OSP 13
+* Vexxhost (private cloud offering): OpenStack based on 16
```

Comment:
This is not exact. Vexxhost is generally offering a recent version of OpenStack, but that is not OSP (so the 16 doesn't mean much).
Checking the current version of keystone running in Vexxhost, I'd say they already upgraded to Ussuri, mapping to OSP 17 if my math is correct.